docs/bpftool-btf.rst

```diff
@@ -35,61 +35,61 @@ BTF COMMANDS
 DESCRIPTION
 ===========
 	**bpftool btf { show | list }** [**id** *BTF_ID*]
-		  Show information about loaded BTF objects. If a BTF ID is
-		  specified, show information only about given BTF object,
-		  otherwise list all BTF objects currently loaded on the
-		  system.
+		Show information about loaded BTF objects. If a BTF ID is
+		specified, show information only about given BTF object,
+		otherwise list all BTF objects currently loaded on the
+		system.
```

Comment:
Here's an example of the changes mentioned above:

- Command syntax without the bold markers (this is valid only for the `DESCRIPTION` where we use description lists, not in the `SYNOPSYS` or command summary)
- Command syntax indented with spaces, here 4 spaces
- Command description indented with spaces, here 4 more spaces
- Command description re-wrapped on 80 characters

```suggestion
    bpftool btf { show | list } [id *BTF_ID*]
        Show information about loaded BTF objects. If a BTF ID is specified,
        show information only about given BTF object, otherwise list all BTF
        objects currently loaded on the system.
```

Reply:
noted. Working on it.

- 1 Tab = 4 spaces for 'Description' & 'Options' Section.
-